clustered/client/src/test/java/org/ehcache/clustered/client/docs/GettingStarted.java

```diff
@@ -229,35 +229,37 @@ public void unknownClusteredCacheExample()
 
     PersistentCacheManager cacheManager1 = cacheManagerBuilderAutoCreate.build(false);
     cacheManager1.init();
+    try {
```

Comment:
I know these are samples, but is there any reason you didn't shift them to try-with-resources as you have in the tests?

Reply:
As discussed given the funky (maybe broken) contract of close I'd rather avoid encouraging the try-with-resources pattern with `.build(false)` cache managers.